pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -204,6 +204,9 @@ func (r *bindingReconciler) reconcile(ctx context.Context, apiBinding *apisv1alp
 		return reconcileStatusContinue, nil
 	}
 
+	clusterName := logicalcluster.From(apiExport)
+	apiBinding.Status.LogicalCluster = clusterName.String()
+
```

Comment:
@ncdc If `APIExportClusterName` is `""`/`logicluster.Name.IsEmpty() == true`, do we want to fall back to the name component of the APIExport's path?

Reply:
It can't be empty by the time you get down here. We have the fallback logic above:
```go
// Get APIExport
	apiExportPath := logicalcluster.NewPath(apiBinding.Spec.Reference.Export.Path)
	if apiExportPath.Empty() {
		apiExportPath = logicalcluster.From(apiBinding).Path()
	}
	apiExport, err := r.controller.getAPIExport(apiExportPath, workspaceRef.Name)
```